styx-common/src/main/java/com/spotify/styx/api/Middlewares.java

```diff
@@ -71,6 +73,8 @@
   private static final String REQUEST_ID = "request-id";
   private static final String X_REQUEST_ID = "X-Request-Id";
 
+  private static final Key<GoogleIdToken> ID_TOKEN_KEY = Context.key("id_token");
```

Comment:
We used to have unused code utilising https://github.com/spotify/apollo/blob/1.x/apollo-api/docs/middleware.md#advanced-creating-custom-context-types . Do you think that would be better compared with using grpc context?

Reply:
RIght, that might be better, not sure.

Reply:
I guess the drawback is it would impact all routes using this middleware.

Reply:
We now use the `authed` middleware.